ingredient/ingredient-kubernetes/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@azbake/ingredient-kubernetes",
+  "description": "Ingredient for deploying Kubernetes configs",
+  "version": "0.0.0",
+  "main": "dist/index.js",
+  "typings": "src/index.ts",
+  "author": "HCHB",
+  "license": "MIT",
+  "scripts": {
+    "upload": "tsc -p . && npm --no-git-tag-version version patch && npm publish --access public",
+    "compile": "tsc -p .",
+    "watch": "tsc -w -p ."
+  },
+  "directories": {
+    "lib": "dist"
+  },
+  "peerDependencies": {
+    "@azbake/core": "0.*",
+    "@azure/ms-rest-nodeauth": "0.9.2"
+  },
+  "devDependencies": {
+    "@azbake/core": "^0.1.44",
+    "@azure/ms-rest-nodeauth": "0.9.2",
+    "@types/node": "^10.12.18"
+  },
+  "dependencies": {
+    "@azbake/ingredient-host-names": "^0.1.44"

```

Comment:
You can't create dependencies on other ingredients. What functionality are you trying to use from this ingredient?